test/ecies.js

```diff
@@ -132,4 +132,31 @@ describe('ECIES', function() {
     }).should.throw('Invalid checksum');
   });
 
+  it('decrypting compressed and uncompressed keys', function() {
+    var secret = 'test';
+
+    // test uncompressed
+    var alicePrivateKey = new bitcore.PrivateKey.fromObject(new Buffer('1fa76f9c799ca3a51e2c7c901d3ba8e24f6d870beccf8df56faf30120b38f360', 'hex'));
```

Comment:
Instantiating the private key for compressed and uncompressed (line 139 and line 151 should be more clear with this:
```js
var alicePrivateKey = new bitcore.PrivateKey.fromObject({
  bn: '1fa76f9c799ca3a51e2c7c901d3ba8e24f6d870beccf8df56faf30120b38f360',
  compressed: false,
  network: 'livenet'
});
```